package/autoyast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Dec  7 12:54:27 CET 2016 - schubi@suse.de
+
+- Cloning Software: install_recommended can be set by the
+  control.xml file (ay_clone_install_recommended_default)
+  Default is true. (Fate#321764)
+- 3.1.151.3.1
```

Comment:
Only for CASP? I mean, does it make sense to add this to the SLE-12-SP2 codebase? Just asking...